<?xml version="1.0" encoding="utf-8"?>

<launch>

	<rosparam file="$(find grasp_learning)/param/box_poses.yaml" command="load"/>

	<node name="box_poses" pkg="grasp_learning" type="box_poses" cwd = "node" respawn="false" output="screen"/>

</launch>